src/develop/masks.h

```diff
@@ -37,7 +37,8 @@ typedef enum dt_masks_type_t
   DT_MASKS_CLONE = 1 << 3,
   DT_MASKS_GRADIENT = 1 << 4,
   DT_MASKS_ELLIPSE = 1 << 5,
-  DT_MASKS_BRUSH = 1 << 6
+  DT_MASKS_BRUSH = 1 << 6,
+  DT_MASKS_NON_CLONE = 1 << 7
```

Comment:
Why not use DT_MASKS_UNMANAGED as this is to avoid the mask to be added into the mask manager. I would at the same time using this bit for the DT_MASKS_CLONE and using only DT_MASKS_UNMANAGED when we check for the type to see if we want it into the mask manager. Seems like it is cleaner and convey what we are trying to do. What do you think?

Reply:
I agree, but then the spots iop should be modified to change the type of the masks and the version incremented.
On the other hand, I still don't know if this will be a new module or an upgrade for the spots, in this last case there will be no need to modify it.

Reply:
I see, sure the spot healing could be ported into the spot removal iop, but it seems handy to me to have the retouch iop with the spot integrated otherwise we will have lot of back and forth between those two iop while working on a portrait. That's to be discussed indeed!